src/panels/lovelace/components/hui-theme-select-editor.ts

```diff
@@ -0,0 +1,65 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import "@polymer/paper-button/paper-button";
+import { TemplateResult } from "lit-html";
+
+import { HomeAssistant } from "../../../types";
+import { fireEvent } from "../../../common/dom/fire_event";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+
+export class HuiThemeSelectionEditor extends hassLocalizeLitMixin(LitElement) {
+  protected hass?: HomeAssistant;
+  private value?: string;
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      value: {},
+    };
+  }
+
+  protected render(): TemplateResult {
+    const themes = ["Backend-selected", "default"].concat(
+      Object.keys(this.hass!.themes.themes).sort()
+    );
+    return html`
+      ${this.renderStyle()}
+      <paper-dropdown-menu
+        dynamic-align
+        @value-changed=${this._changed}
+      >
+        <paper-listbox
+          slot="dropdown-content"
+          selected=${this.value || "Backend-selected"}
```

Comment:
`.selected`

Reply:
It's a string. I thought you said if it was a string not using property was better in the long run?

Reply:
Ex. HA-icon `icon` property

Reply:
not using a property is better if it's not dynamic. Because setting a property requires a dynamic slot `${}` and so will be re-rendered on each change.

`<ha-icon icon="bla"></ha-icon>` has no dynamic slots, gets not re-rendered ever

Reply:
Understood